magma/wire_container.py

```diff
@@ -1,6 +1,13 @@
+import logging
+from magma.config import config, EnvConfig
```

Comment:
nit + I think we've used py_logging as standard else where to avoid confusing with internal `magma.logging` module

```suggestion
import logging as py_logging

from magma.config import config, EnvConfig
```